rosbag2_performance/rosbag2_performance_benchmarking/src/benchmark_publishers.cpp

```diff
@@ -87,19 +85,29 @@ class BenchmarkPublishers : public rclcpp::Node
     }
 
     const std::string node_name(get_fully_qualified_name());
+    const auto initial_time = std::chrono::high_resolution_clock::now() + std::chrono::seconds(1);
```

Comment:
@carlossvg I would advise to rename `initial_time` to `when_to_start` for clarity

Reply:
done